app/components/subheader/bootstrap.yml

```diff
@@ -1,3 +1,7 @@
 _components:
   subheader:
-    text: ''
+    text: ""
+    type: "h4"
+    css: ""
+    link: ""
+    iconcss: "link"
```

Comment:
If this value is not editable then you can just hardcode it into the template rather than declaring it here and assigning it in the `model.js`